### PR TITLE
Expose dependency comment content

### LINKED ISSUE #250: Supporting action output
👋 Hi! My team has been using this action, and we were wondering if it's possible to get the summary output from the action to use in subsequent workflow steps. A fake example:
image

We'd like to have this summary output available as output from the action itself, so that we can do things like post it as a comment on the PR that corresponds to the workflow run, notify people in Slack, etc. Is that supported today and I'm just missing it? If not, I would be happy to put together a pull request to support this.

### PR DESCRIPTION
**Pull request by jsoref**
Closes #250

### CODE REVIEW

**Review by febuiles — COMMENTED** [review]
@jsoref thanks for this, it looks helpful, and code-wise I think we're good. 

Before merging: can you add an example of how this would be used in https://github.com/actions/dependency-review-action/blob/main/docs/examples.md?

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # dependency-review-action
 
 This action scans your pull requests for dependency changes, and will
-raise an error if any vulnerabilities or invalid licenses are being introduced. The action is supported by an [API endpoint](https://docs.github.com/en/rest/reference/dependency-graph#dependency-review) that diffs the dependencies between any two revisions on your default branch.
+raise an error if any vulnerabilities or invalid licenses are being introduced. The action is supported by an [API endpoint](https://docs.github.com/rest/reference/dependency-graph#dependency-review) that diffs the dependencies between any two revisions on your default branch.
```
Comment (jsoref) [inline README.md:4]: I could split these other changes into additional PRs if desired, but I was in the neighborhood...

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
 
 You can see the results on the job logs:
 
-<img width="854" alt="Screen Shot 2022-03-31 at 1 10 51 PM" src="https://user-images.githubusercontent.com/2161/161042286-b22d7dd3-13cb-458d-8744-ce70ed9bf562.png">
+<img width="854" alt="GitHub workflow run log showing Dependency Review job output" src="https://user-images.githubusercontent.com/2161/161042286-b22d7dd3-13cb-458d-8744-ce70ed9bf562.png">
 
 or on the job summary:
 
-<img src="https://user-images.githubusercontent.com/7847935/182871416-50332bbb-b279-4621-a136-ca72a4314301.png">
+<img alt="GitHub job summary showing Dependency Review output" src="https://user-images.githubusercontent.com/7847935/182871416-50332bbb-b279-4621-a136-ca72a4314301.png">
```
Comment (jsoref) [inline README.md:14]: Note that the picture here is really out of date, it includes a typo that was fixed in 2022: https://github.com/actions/dependency-review-action/commit/dfcdb87cb32dee2e261aea8cb01fe9b0051e1917

File: docs/examples.md [hunk 3]
```diff
@@ -164,6 +164,37 @@ jobs:
           comment-summary-in-pr: always
 ```
 
+## Getting the results of the action in a later step
+
+Using the `comment-content` output you can get the results of the action in a workflow step.
```
Comment (jsoref) [inline docs/examples.md:169]: This precise workflow is untested.

The workflow I used is here:
https://github.com/check-spelling-sandbox/dependency-review-action-0/blob/657ea37667b5afdb8c94c4197272ba19adbfa0af/.github/workflows/dependency-review.yml

I _believe_ they're functionally equivalent, but someone should really carefully review (or better, test).

**Comment by febuiles** [comment]
@jsoref your branch won't allow me to push changes (there's a checkbox for this on PR creation!), so I opened a new PR here to tweak and merge: https://github.com/actions/dependency-review-action/pull/696. Closing this one. 

Thanks a ton for the extra doc fixes!

**Comment by jsoref** [comment]
Hmm... I wonder if I have a pref set to block that. I thought I _usually_ allowed them...

**Pull request closed without merging**